README.md

```diff
@@ -9,15 +9,14 @@ This repository holds the source for this Implementation Guide, part of the fami
 
 ### What does it do?
 
--  TBC
-
+-   DNS-DS provides a simple and flexible method to discover NMOS RDS Services.
 ### Why does it matter?
 
-- TBC
+- The NMOS RDS provides all information about NMOS nodes, devices, APIs. It must be located in order for a NNMOS component to register itself for discovery by other NMOS components or for discovering other NMOS components on a network. 
```

Comment:
Not sure how well known "RDS" is, so "NMOS's registration and discovery system" better?

Reply:
```suggestion
- An NMOS Registration and Discovery System (RDS) provides information about NMOS Nodes, APIs, Devices, and other resources. An NMOS Node needs to find the RDS to register itself and its resources so that that they can be discovered by other NMOS Nodes and clients.

Reply:
LGTM